planning/behavior_path_planner/include/behavior_path_planner/scene_module/goal_planner/goal_planner_module.hpp

```diff
@@ -277,6 +291,17 @@ class GoalPlannerModule : public SceneModuleInterface
   // output setter
   void setOutput(BehaviorModuleOutput & output);
   void setStopPath(BehaviorModuleOutput & output);
+
+  /**
+   * @brief Sets a stop path in the current path based on safety conditions and previous paths.
+   *
+   * This function sets a stop path in the current path. Depending on whether the previous safety
+   * judgement against dynamic objects were safe or if a previous stop path existed, it either
+   * generates a new stop path or uses the previous stop path.
+   *
+   * @param output BehaviorModuleOutput
+   */
+  void setStopPathInCurrentPath(BehaviorModuleOutput & output);
```

Comment:
not `In`?

Reply:
and this function does not change the current path.
just change output and status_.prev_stop_path_after_approval.

Reply:
In my understanding, both @param [in] and @param is acceptable, but not so sure....

Reply:
sorry, talked about the function name `setStopPathInCurrentPath`

Reply:
From?

Reply:
changed in https://github.com/autowarefoundation/autoware.universe/pull/4927/commits/c288508a6ab7c92c67b5fbc37d8411a7516a4580